File: code/src/main/java/service/SuperService.java
package service;

import model.User;

public class SuperService {
    public ServiceUser serviceUser;
    public ServicePerformance servicePerformance;
    public ServiceTicket serviceTicket;
    private User user;

    public SuperService(ServiceUser serviceUser, ServicePerformance servicePerformance, ServiceTicket serviceTicket, User user) {
        this.serviceUser = serviceUser;
        this.servicePerformance = servicePerformance;
        this.serviceTicket = serviceTicket;
        this.user = user;
    }

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }
}

File: MusicFestivalMPP/service/SuperService.cs
using MusicFestivalMPP.model;

namespace MusicFestivalMPP.service;

public class SuperService
{
    public ServiceUser ServiceUser;
    public ServicePerformance ServicePerformance;
    public ServiceTicket ServiceTicket;
    public User User { get; set; }

    public SuperService(ServiceUser serviceUser, ServicePerformance servicePerformance, ServiceTicket serviceTicket, User user) {
        this.ServiceUser = serviceUser;
        this.ServicePerformance = servicePerformance;
        this.ServiceTicket = serviceTicket;
        this.User = user;
    }
}